src/EditorFeatures/Core/FindUsages/ClassifiedSpansAndHighlightSpanFactory.cs

```diff
@@ -44,6 +49,46 @@ internal static class ClassifiedSpansAndHighlightSpanFactory
                 documentSpan.Document, documentSpan.SourceSpan, cancellationToken).ConfigureAwait(false);
         }
 
+        public static ClassifiedTextElement BuildClassifiedTextElementForClassifiedSpans(IEnumerable<ClassifiedSpan> classifiedSpans, ITextSnapshot snapshot, CancellationToken cancellationToken)
```

Comment:
does this method use anything from this factory? otherwise, this seems not the place to put this in. when I said refactoring, I actually meant to share code not just move method to this factory.


if this method can't share any code with existing code and static. this should be just extension method like this - http://index/?query=toInlines&rightProject=Microsoft.CodeAnalysis.EditorFeatures.Wpf&file=WpfClassificationExtensions.cs&line=30 - I didn't mean to put this method to the above file, I just mean extension method seems more useful for people later.